interfaces/IBF-dashboard/src/global.scss

```diff
@@ -318,3 +318,8 @@ app-tooltip {
 .font-size-12 {
   font-size: 12px;
 }
+
+a {
```

Comment:
Use tailwind's [underline](https://tailwindcss.com/docs/text-decoration-line#underling-text) and [colour](https://tailwindcss.com/docs/colors#customizing-your-colors) instead.

Reply:
does this require me to add class="underline" (and something about colour) separately to all <a> elements in the codebase? That seems unhandy. Am I missing something?